iotcore/src/main/java/com/google/android/things/iotcore/ConnectionParams.java

```diff
@@ -284,6 +287,7 @@ private ConnectionParams(
         mTelemetryTopic = "/devices/" + mDeviceId + "/events";
         mDeviceStateTopic = "/devices/" + mDeviceId + "/state";
         mConfigurationTopic = "/devices/" + mDeviceId + "/config";
+        mCommandsTopicPrefix = "/devices/" + mDeviceId + "/commands";
```

Comment:
how about mCommandsTopic? I understand why you choose mCommandsTopicPrefix, but I suggest that it be consistent with mTelemetryTopic which also allows subfolder.

Reply:
okay, I will change it